experiments/ClimaCore/sea_breeze/atmos_rhs.jl

```diff
@@ -213,6 +212,27 @@ function atm_rhs!(dY, Y, params, t)
     return dY
 end
 
+# Atmos Simulation - later to live in ClimaAtmos
+struct AtmosSimulation <: ClimaCoupler.AbstractAtmosSimulation
+    integrator::Any
```

Comment:
I think access to `Y_init` is useful for debugging. Actually is `Y_init = integrator.sol.u[1]` with how we currently use the `step!` function?

Reply:
> Actually is `Y_init = integrator.sol.u[1]` with how we currently use the `step!` function?

I think so - as long as we save the solution after the init then we would have access to this. We could make some helper functions to grab the initial conditions though!